paws.common/NEWS.md

```diff
@@ -2,6 +2,8 @@
 
 * Fix Content-Md5 being modified by user
 * Add logging system to help with debugging paws issues
+* Add support for `AssumeRoleWithWebIdentity` (#477)
+* Clarify that `AWS_CREDENTIAL_EXPIRATION` is optional when using `AWS_SESSION_TOKEN` (#549)
```

Comment:
feel free to add yourself to this i.e. (thanks to @fh-mthomson for contribution)

Reply:
Added and resolved merge conflicts :)